### ISSUE TITLE
Add minimal CLI

### ISSUE THREAD

**Issue by ckunki** [exasol/bucketfs-python#33]
In the past numerous attempts have been made to mitigate interaction with Exasol's BucketFS:

* :heavy_check_mark: [bucketfs-python](https://github.com/exasol/bucketfs-python): Python, active, used for tests automation of various python projects.
* :heavy_check_mark: [bucketfs-java](https://github.com/exasol/bucketfs-java): Java, used for tests automation of various java projects.
* :question: [bucketfs-client](https://github.com/exasol/bucketfs-client): Java, limited functionality, currently not develeoped actively
* :x: [bucketfs-explorer](https://github.com/exasol/bucketfs-explorer): Java, GUI application, archived, deprecated
  Currently contained in official documentation (see DOC-2221)
* :x: [bucketfs-utils-python](https://github.com/exasol/bucketfs-utils-python/): depecated, superseeded by bucketfs-python
* :question: shell functions for bash based on CURL requests: only for power users, limited functionality, see below

```shell
function bucketfs-password() {
    if [ -z "$1" ]; then
       echo "usage: bucketfs-password <container>"
       return 1
    fi
    BUCKETFS_PASSWORD=$(
	docker exec -it $1 \
	       grep WritePass /exa/etc/EXAConf \
	    | sed -e "s/.* = //" \
	    | tr -d '\r' \
	    | base64 -d)
}

function bucketfs-upload() {
    if [ -z "$1" ]; then
       echo "usage: bucketfs-upload <file> [path/in/bucket-fs]"
       return 1
    fi
    if [ -z "$BUCKETFS_PASSWORD" ]; then
       echo "Please set environment variable BUCKETFS_PASSWORD"
       return 1
    fi
    A=2580 # port
    B=$(echo default/$2 | sed -e 's/\/$//') # path
    curl -v -X PUT -T $1 http://w:$BUCKETFS_PASSWORD@localhost:$A/$B/$1
}
```

Still users struggle and need help from time to time.  The current ticket therefore requests to create a minimal CLI solution
* convenient and easy to use
* acceptable usability with sufficient functionality and at least some guidance for unexperienced users
* lightweight with minimal footprint and prerequisites, e.g. frameworks and installations

In summary bucketfs-python seems to be the best candidate. It is used anyway for test automation of various python projects.  Currently issue #4 requires to enhance bucketfs-python to list the contents of a folder in BucketFS which could be another building block in order to provide a minimal CLI with very limited effort.

**Comment by ckunki** [comment]
Closed in favor of https://github.com/exasol/bucketfs-client/issues/13
